lib/credentials.js

```diff
@@ -70,6 +83,12 @@ Credentials.prototype._expand = function() {
     this.requestPubKey = requestDerivation.publicKey.toString();
   }
   var network = WalletUtils.getNetworkFromXPubKey(this.xPubKey);
+  if (typeof this.externalSource == "string") {
+    var xPrivKey = new Bitcore.HDPrivateKey(network);
```

Comment:
We are planning some changes related to the `request key`. It is not final yet but it should include the following changes:
- Request priv/pub key pairs do not need to be derived from the master xpriv.
- At any time, copayers will be able to generate a new request key and send it to BWS (which in turn will make it available to all other copayers). The only requirement is that the new request pub key be signed with the master xpriv.
We think this will make it possible to restore a wallet exclusively from a master xpriv key.

As for your code, would you please change this to generate a simple private/public key pair (no HD derivation)?